tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sources/AvroKafkaSourceTest.java

```diff
@@ -117,8 +117,8 @@ public void startKafkaContainers(PulsarCluster cluster) throws Exception {
         sourceConfig.put("topic", kafkaTopicName);
         sourceConfig.put("valueDeserializationClass", "io.confluent.kafka.serializers.KafkaAvroDeserializer");
         sourceConfig.put("consumerConfigProperties",
-                ImmutableMap.of("schema.registry.url", getRegistryAddressInDockerNetwork())
-        );
+                ImmutableMap.of("schema.registry.url", getRegistryAddressInDockerNetwork()));
+        sourceConfig.put("copyHeadersEnabled", true);
```

Comment:
The integ test is failing because the `kafka-avro-console-producer` doesn't respect the headers.pars option. Will disable the test for now and add it as needed. 

https://github.com/confluentinc/schema-registry/issues/2390